src/inspect.rs

```diff
@@ -189,3 +209,106 @@ impl MetadataPrinter for MetadataPrettyPrinter {
         self.print_metadata_usage(metadata)
     }
 }
+
+fn get_signatures_printer(output_type: OutputType) -> Box<dyn SignaturesPrinter> {
```

Comment:
While using the trait system is correct, I feel like this is kinda an overkill in this scenario and I'm not so sure about that being idiomatic.

I think this could be solved by defining `SignaturePrinter` to be an `enum` with two values: `Yaml` and `Pretty`. Then the `print` method would behave differently based on the type of the enum.

@ereslibre what do you think? I'm interested to hear your opinion about the idiomatic part. I do not intend to nitpick on this code written by @raulcabello :)

Reply:
Yes, I agree. Even something like `PrinterOutput`, `OutputFormat` or something similar, I think is closer to the intent of the parameter, but I think is better to not use traits in a similar way as we would use go interfaces, with Rust we have a more powerful type system :)

Reply:
I'm still fairly new to rust and I'd love to write more idiomatic code. So this feedback is very appreciated! 
Should I also refactor the [MetadataPrinter](https://github.com/kubewarden/kwctl/blob/main/src/inspect.rs#L44-L65)?
Maybe it makes more sense to merge MetadataPrinter and SignaturesPrinter into one Printer?

Reply:
I think having two printers is fine, given they are focused on printing different types of information.

For uniformity I would refactor the `MetadataPrinter` as well, yes.

Reply:
done with enums https://github.com/kubewarden/kwctl/pull/190/commits/2a554bd961017abb09befb5ac28edba4dd82717d